app/javascript/packages/document-capture-polling/index.ts

```diff
@@ -91,7 +91,7 @@ export class DocumentCapturePolling {
   onComplete({ result, redirect }: { result: ResultType; redirect?: string }) {
     this.trackEvent('IdV: Link sent capture doc polling complete', {
       isCancelled: result === ResultType.CANCELLED,
-      isThrottled: result === ResultType.THROTTLED,
+      isRateLimited: result === ResultType.RATE_LIMITED,
```

Comment:
@R-Ornelas one more throttle-related analytics change. 'IdV: Link sent capture doc polling complete' argument is now isRateLimited, formerly isThrottled.